StantUiIosLibrary/Classes/Tables/ContactsTableViewWithCollapsedSearchBar/Extensions/ContactsViewShowAndHideSearchTableViewExtension.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ContactsViewShowAndHideSearchTableViewExtension.swift
+//  Pods-StantUiIosLibraryDemo
+//
+//  Created by Renato Vieira on 02/12/19.
+//
+
+import UIKit
+
+extension ContactsTableViewWithCollapseSearchBar: ContactsAndHideSearchTableViewDelegate {
+    public func collapseSearchView() {
+        guard let searchView = searchView, let tableView = tableView else { return }
+        self.removeAllConstraints()
+               
+        UIView.animate(withDuration: 0.2, delay: 0, options: [.transitionCrossDissolve], animations: {
+                searchView.isHidden = true
+                   tableView.removeAllConstraints()
+                   tableView.frame     = CGRect(x: tableView.frame.minX, y: 0, width: self.frame.width,
```

Comment:
Remove extra spaces and break CGRect parameters.

Reply:
Done